sub-packages/bionemo-esm2/src/bionemo/esm2/run/config_models.py

```diff
@@ -223,6 +223,7 @@ def model_validator(self, global_cfg: MainConfig) -> MainConfig:
         ), "Must set variable_seq_lengths to True when min_seq_length != max_seq_length under pipeline or tensor parallelism."
         return global_cfg
 
-    def model_class(self) -> Type[ESM2Config]:
+    @classmethod
+    def model_class(cls) -> Type[ESM2Config]:
```

Comment:
I'd say either make it a class method and return `cls` or static and return the one class? Right now there's no change in terms of correct usage of class method vs instance method. Static would be the lowest featured method type that still works here.

Reply:
oh good callout!